programs/bpf_loader/src/lib.rs

```diff
@@ -214,6 +221,36 @@ pub fn process_instruction(
     Ok(())
 }
 
+pub fn process_instruction(
+    program_id: &Pubkey,
+    keyed_accounts: &[KeyedAccount],
+    instruction_data: &[u8],
+    invoke_context: &mut dyn InvokeContext,
+) -> Result<(), InstructionError> {
+    process_instruction_general(
+        program_id,
+        keyed_accounts,
+        instruction_data,
+        invoke_context,
+        false,
+    )
+}
+
+pub fn process_instruction_jit(
```

Comment:
Doesn't need to be `pub` right?

Reply:
Still needs it as it is used by a macro in a different file.

Reply:
ah, yep. thanks